railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -17,6 +17,7 @@ Rails.application.configure do
   # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
   # Requires an encryption key in `ENV["RAILS_MASTER_KEY"]` or
   # `config/secrets.yml.key`.
+  # config.secrets_cipher = "aes-128-gcm"
```

Comment:
I'm all to add a config for this but I don't think we should promote it by default.